app/views/users/show.html.erb

```diff
@@ -1 +1,26 @@
-<h1><%= @user.name %></h1>
\ No newline at end of file
+<a href="<%= user_posts_path(@user) %>">
+    <button id="back-btn">⬅</button>
+</a>
+<div class="user-box">
+    <%= render partial: 'shared/user_photo', locals: { user: @user } %>
+    <%= render partial: 'shared/user_details', locals: { user: @user } %>
+</div>
+<div class="bio">
+    <h3>Bio</h3>
+    <p>
+        <%= @user.bio %>
+    </p>
+</div>
+<% @user.posts.each do |post| %>
+    <div class="post bio">
+        <h3>Post <%= raw "#" + post.id.to_s %></h3>
```

Comment:
- [ Required ] I think it would be nice if you make it possible for a user to navigate to the `posts_show` page when a post title is clicked on. Currently, there is no link on the `users_show` page that can be used to visit a particular `post's show` page. Please see the example code below for a suggestion of how to achieve this.

## Example code
```ruby
<% @user.posts.each do |post| %>
    <a href="<%= user_post_path(@user, post) %>">
    <div class="post bio">
        <h3>Post <%= raw "#" + post.id.to_s %></h3>
        <p><%= post.text %></p>
        <div class='post-interactions'>
            <span>Comments: <%= post.comments_counter %>,</span>
            <span>Likes: <%= post.likes_counter %></span>
        </div>
    </div>
    </a>
<% end %>
```